e2e/apprunner/front-end/main.go

```diff
@@ -40,11 +41,11 @@ func SimpleGet(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
 // This test assumes the backend app is called "back-end". The 'service-discovery' endpoint
 // of the back-end service is unreachable from the LB, so the only way to get it is
 // through service discovery. The response should be `back-end-service-discovery`
-func ServiceDiscoveryGet(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
+func ServiceDiscoveryGet(w http.ResponseWriter, req *http.Request) {
 	endpoint := fmt.Sprintf("http://back-end.%s/service-discovery/", os.Getenv("COPILOT_SERVICE_DISCOVERY_ENDPOINT"))
 	resp, err := http.Get(endpoint)
 	if err != nil {
-		log.Printf("🚨 could call service discovery endpoint: err=%s\n", err)
+		log.Printf("🚨 could call service discovery endpoint: err=%s", err)
```

Comment:
tiny q: why do we not want the new line anymore?

Reply:
`log.Printf` adds a newline already!